### PR TITLE
Us1 b

### PR DESCRIPTION
**Pull request by fturello**

### CODE REVIEW

**Comment by fturello** [comment]
@lchastanet please

**Review by Julien879465 — APPROVED** [review]

**Review by antoine-droid — APPROVED** [review]
🫠

**Review by MrRobo1 — APPROVED** [review]

File: backend/src/middleware/verifyUser.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const model = require("../models");
+
+const verifyUser = (req, res, next) => {
+  // FavoriteManager.findAllByUserId(req.userId);
```
Comment (lchastanet) [inline backend/src/middleware/verifyUser.js]: Code commented

File: backend/src/router.js [hunk 2]
```diff
@@ -4,14 +4,18 @@ const router = express.Router();
 
 const { hashingPassword } = require("./middleware/auth");
 const { login, logout } = require("./controllers/authController");
+const rolesCheck = require("./middleware/rolesCheck");
+// const signUp = require("./controllers/signUpController");
```
Comment (lchastanet) [inline backend/src/router.js]: Code commented

File: backend/src/router.js [hunk 3]
```diff
@@ -4,14 +4,18 @@ const router = express.Router();
...
 
 const itemControllers = require("./controllers/itemControllers");
 const videoControllers = require("./controllers/videoControllers");
 const favoriteControllers = require("./controllers/favoriteControllers");
 const categoryControllers = require("./controllers/categoryControllers");
-
+const authorization = require("./middleware/authorization");
 const userControllers = require("./controllers/userControllers");
 
+// const verifyUser = require("./middleware/verifyUser");
```
Comment (lchastanet) [inline backend/src/router.js:15]: Code commented

File: backend/src/router.js [hunk 4]
```diff
@@ -43,6 +54,11 @@ router.get("/favorites", favoriteControllers.browse);
 router.get("/favorites/:id", favoriteControllers.read);
 router.put("/favorites/:id", favoriteControllers.edit);
 router.post("/favorites", favoriteControllers.add);
-router.delete("/favorites/:id", favoriteControllers.destroy);
+router.delete(
+  "/favorites/:id",
+  authorization,
+  // verifyUser,
```
Comment (lchastanet) [inline backend/src/router.js:51]: Code commented

File: frontend/src/components/Footer.jsx [hunk 5]
```diff
@@ -1,15 +1,34 @@
+/* eslint-disable react/prop-types */
```
Comment (lchastanet) [inline frontend/src/components/Footer.jsx]: Ah oui carrément

File: frontend/src/pages/Favorites.jsx [hunk 6]
```diff
@@ -6,33 +6,48 @@ import styles from "../styles/Favorites.module.scss";
 
 export default function Favorites() {
   const [favorites, setFavorites] = useState([]);
+
+  // console.log(favorites);
```
Comment (lchastanet) [inline frontend/src/pages/Favorites.jsx]: Code commented

**Review by lchastanet — CHANGES_REQUESTED** [review]

File: backend/src/router.js (hunk 3 above)
Comment (fturello) [reply backend/src/router.js:15]: i need to keep it for now

File: backend/src/router.js (hunk 4 above)
Comment (fturello) [reply backend/src/router.js:51]: i need to comment it for now because it breaks my code

**Review by lchastanet — APPROVED** [review]

**Pull request merged**